Classes/Common/Indexer.php

```diff
@@ -144,7 +145,7 @@ public static function add(Document $document, DocumentRepository $documentRepos
                 }
                 // Commit all changes.
                 $updateQuery = self::$solr->service->createUpdate();
-                $updateQuery->addCommit();
+                $updateQuery->addCommit($softCommit, null, null);
```

Comment:
```suggestion
                $updateQuery->addCommit($softCommit);
```
We don't need the second and third parameter, since `null` is their default value, right?